src/schemathesis/specs/openapi/examples.py

```diff
@@ -0,0 +1,104 @@
+from typing import Any, Dict, List, Optional
+
+from hypothesis.strategies import SearchStrategy
+
+from ..._hypothesis import PARAMETERS, _get_case_strategy, prepare_strategy
+from ...models import Case, Endpoint
+
+
+def get_param_examples(endpoint_def: Dict[str, Any], examples_field: str) -> List[Dict[str, Any]]:
+    all_parameter_examples = []
+    for param in endpoint_def.get("parameters", []):
+        if examples_field in param:
+            # add value to list of params with examples
+            param_type = {
+                "body": "body",
+                "cookie": "cookies",
+                "header": "headers",
+                "path": "path_parameters",
+                "query": "query",
+            }.get(param["in"])
+            assert param_type in PARAMETERS
```

Comment:
My thought is that, if the keywords in `PARAMETERS` change over time but this mapping is not updated, the assertion would help catch the bug.

Reply:
I think we can move this check to the import time - in this case it will be visible immediately if this mapping is up-to-date. Then this code might be rewritten as a list comprehension. Also, I am not sure but we might have it in multiple places (need to check this) so we can reuse if it will be available globally

Reply:
Yes, the mapping is also used in `specs/openapi/links`. I will make the mapping global, removes the assertion above, and use the mapping in `links.py` and `examples.py`.